### DISCUSSION TITLE
All cameras offline in homekit after 1 -2 min

### DISCUSSION THREAD

**Discussion by revil-O** [koush/scrypted#863] [Q&A]
hi, the newest strange thing i face with my camera‘s is that all of them (2 ring, 1 foscam rtsp) disappear in homekit after 2 mins.
a manual reload of the homekit plugin in scrypted webui brings tem back for additional  1 -2 minutes. —> loop of behaviour.

this behaviour starts 2 days ago. config wasn‘t changed by myself.

Versions:
iOS: 16.5
scrypted: 0.23.0
homekit@1.2.27
ring@0.0.110
rtsp@0.0.53

need more details? kindly request :)

regards olly

**Comment by MediumFidelity** [comment]
If you don't reload the HomeKit plug in the Scrypted web UI, do the cameras come back on their own after a while?

**Reply by MediumFidelity** [reply]
(for my issue, I don't see Amcrest cameras in your setup)

I ended up switching from the Amcrest plugin to the plain ONVIF one for my cameras.  They ran overnight with no cameras dropping offline.
